frank/fit.py

```diff
@@ -414,6 +425,16 @@ def output_results(u, v, vis, weights, sol, iteration_diagnostics, model):
 
     figs, axes = [], []
 
+    if model['plotting']['deprojec_plot']:
+        up, vp, visp = geom.apply_correction(u, v, vis)
+        deproj_fig, deproj_axes = make_figs.make_deprojection_fig(u, v, up, vp, vis, visp,
+                                                         model['plotting']['force_style'],
```

Comment:
hm I was trying to keep it so that the functions in `make_figs.py` depend only on functions in `plot.py` and `utilities.py`, e.g., none of the other functions in `make_figs.py` take a geometry object as an input. I'd personally prefer it this way because then the function in `make_figs.py` really just has a single purpose: to make a figure, not to also perform a deprojection. 

None of the other functions in `make_figs.py` make a plot of `u` vs. `v`, so this function is unique. 

Does that seem reasonable? I can pass in the geometry object if you think that's ideal though.

Reply:
It is certainly harder for users to call this function incorrectly if the arguments are u, v, geom.

Reply:
Moved this inside `make_deprojection_fig` in 7cd5d8a